plugins/linenumbers.lua

```diff
@@ -37,46 +45,47 @@ function DocView:draw_line_gutter(line, x, y, width)
     return lh
   end
 
-  if config.plugins.linenumbers.relative then
-    local color = style.line_number
-    local local_idx = line
+  local color = style.line_number
 
-    for _, line1, _, line2 in self.doc:get_selections(true) do
-      if line >= line1 and line <= line2 then
-        color = style.line_number2
-        break
-      end
-    end
-
-    local l1 = self.doc:get_selection(false)
-    if line == l1 then
+  for _, line1, _, line2 in self.doc:get_selections(true) do
+    if line == line1 then
       color = style.line_number2
-      local_idx = 0
-    else
-      local_idx = math.abs(line - l1)
+      break
     end
+  end
 
-    common.draw_text(
-      self:get_font(),
-      color, local_idx, "right",
-      x + style.padding.x,
-      y,
-      width, lh
-    )
-  else
-    return draw_line_gutter(self, line, x, y, width)
```

Comment:
You should probably keep this.

Reply:
Even though this serves the same purpose?:

```lua
.
.
.
elseif not config.plugins.linenumbers.relative then
    local_idx = line
```

Reply:
It's probably fine to add something like:
```lua
if not config.plugins.linenumbers.relative and not config.plugins.linenumbers.hybrid then
  return draw_line_gutter(self, line, x, y, width)
end
```
near the beginning of the function.

Reply:
Added and checked by trying on lite-xl